c/dec/decode.c

```diff
@@ -16,10 +16,10 @@
 #include "../common/shared_dictionary_internal.h"
```

Comment:
can the "../" cases also be eliminated?

Reply:
Not sure. Current relative path scheme fits Bazel, CMake, Autotools, plain make and python disttools. Will check if we could reliably specify build root to all those build systems.